cmds/task/utils.go

```diff
@@ -35,3 +35,11 @@ func executeHelperE(f Executor) func(*cobra.Command, []string) error {
 		return f(creds, args, cmd.OutOrStdout(), cmd.Flags())
 	}
 }
+
+func stringFlagHelper(flagset *pflag.FlagSet, flag string) string {
+	val, err := flagset.GetString(flag)
+	if err != nil {
+		panic(fmt.Errorf("error: error encountered getting the value of %s: %v", flag, err))
```

Comment:
nit: if we chain this error, we're gonna end with way too many times the word 'error' in a row!